semgrep/semgrep/util.py

```diff
@@ -111,6 +114,22 @@ def with_color(color: str, text: str, bold: bool = False) -> str:
     return click.style(text, fg=color)
 
 
+def terminal_wrap(text: str) -> str:
+    from shutil import get_terminal_size
+    import textwrap
```

Comment:
Do these imports take longer? should we just do them at top level?

Reply:
util is imported on CLI load, so we should minimize its top-level imports